src/tutorials/simple-service-provider.md

```diff
@@ -394,7 +394,15 @@ The JSON.stringify our the data when passing to the `send()` function, because t
 
 * `displayJsonSend(message)` - Will show the message in the "Activity Log" section of the UI.
 
-8. Replace the current `index.html` with the following, to reflect our output on the UI:
+8. We'll take care of the styling before we move onto building our UI. Here we can grab the `assets` folder from this [link]('https://github.com/nymtech/developer-tutorials/tree/main/Simple%20Service%20Provider%20Tutorial/User%20Client/assets'). From here, were just going to drag it into the 'User Client' folder. Once we have that folder in, we can move onto filling out our next file, the `index.html` page to apply these styles.
+
+The easiest way to get the `/assets` folder form this repository is to visit [this Github directory downloader](https://download-directory.github.io/) and enter the link of the folder specified above. From there, the tool will download that specific folder from the example repository into your computers default downloads directory. Drag the `/assets` folder from that downloaded into the `/src` directory of your User Client folder.
```

Comment:
why wouldn't people just fork the repo and copy it in themselves? Seems unnecessary additional step here.